.github/workflows/release-sims.yml

```diff
@@ -1,39 +0,0 @@
-name: Release Sims
-# Release Sims workflow runs long-lived (multi-seed & large block size) simulations
-# This workflow only runs on a pull request when the branch contains rc** (rc1/vX.X.x)
-on:
-  pull_request:
-    branches:
-      - "rc**"
```

Comment:
this would run when we would be doing rcs that is why it never runs.

Reply:
Yes, but we don't name rc branches that way. It didn't run since 2 years.